### DISCUSSION TITLE
4 new things

### DISCUSSION THREAD

**Discussion by tutueldias** [HbmMods/Hbm-s-Nuclear-Tech-GIT#749] [Polls]
add 4 things the first a military base military agents the radiation of the black cloud expands depending on where the cloud goes depending on the wind

**Comment by 70000hp** [comment]
we have the pentagon's top cryptologists trying to decrypt what the hell are you talking about

**Reply by tutueldias** [reply]
esplicame

**Reply by Arthurp-che** [reply]
I think he means the black cloud from the molten RBMK
